lib/index.d.ts

```diff
@@ -361,7 +361,14 @@ declare namespace Realm.Sync {
         expiresAt?: Date;
     }
 
-    type ErrorCallback = (message?: string, isFatal?: boolean, category?: string, code?: number) => void;
+    class SyncError {
```

Comment:
I think this should be just an interface.

Reply:
right will do